diff --git a/ports/CB030/BOOTFILE/dev.bl b/ports/CB030/BOOTFILE/dev.bl
index 9f49df6..1530481 100644
--- a/ports/CB030/BOOTFILE/dev.bl
+++ b/ports/CB030/BOOTFILE/dev.bl
@@ -43,7 +43,7 @@ LOCAL CMDS/BOOTOBJS/t1
 # RBF subsystem, drivers and descriptors
 MWOS OS9/68000/CMDS/BOOTOBJS/rbf
 #MWOS 68000/CMDS/BOOTOBJS/ram
-LOCAL CMDS/BOOTOBJS/cfide
+#LOCAL CMDS/BOOTOBJS/cfide
 LOCAL CMDS/BOOTOBJS/dd
 LOCAL CMDS/BOOTOBJS/c0
 LOCAL CMDS/BOOTOBJS/c0_fmt
diff --git a/ports/CB030/BOOTFILE/diskboot.bl b/ports/CB030/BOOTFILE/diskboot.bl
index 9acfea1..99c013d 100644
--- a/ports/CB030/BOOTFILE/diskboot.bl
+++ b/ports/CB030/BOOTFILE/diskboot.bl
@@ -44,7 +44,7 @@ LOCAL CMDS/BOOTOBJS/t1
 
 # RBF subsystem, CompactFlash driver and descriptors
 MWOS OS9/68000/CMDS/BOOTOBJS/rbf
-LOCAL CMDS/BOOTOBJS/cfide
+#LOCAL CMDS/BOOTOBJS/cfide
 LOCAL CMDS/BOOTOBJS/dd
 LOCAL CMDS/BOOTOBJS/c0
 LOCAL CMDS/BOOTOBJS/c0_fmt
diff --git a/ports/CB030/makefile b/ports/CB030/makefile
index 5ea88bf..aa74575 100644
--- a/ports/CB030/makefile
+++ b/ports/CB030/makefile
@@ -31,7 +31,7 @@ TOUCH		@= os9touch
 
 # subdirectories to build, order is important
 DIRS		+= INIT
-DIRS		+= RBF
+#DIRS		+= RBF
 DIRS		+= SCF
 DIRS		+= SYSMOD
 DIRS		+= BOOTFILE
